README.md

```diff
@@ -1,4 +1,46 @@
 event-emitter
 =============
 
-Event Emitter code
+## Emitter creation
+The easiest way to create an emmiter instance is via `com.metamx.emitter.core.Emitters.create` method.
+This method will create one of the predefined emitters or your own implementation based on the properties provided.
+
+## Emitter types
+
+### Logging emitter
+When properties contains `com.metamx.emitter.logging` property set then an instance of `LoggingEmitter` will be created.
+For more details on `LoggingEmitter` configuration please refer `com.metamx.emitter.core.Emitters.makeLoggingMap()` 
+
+### Http emitter
+When properties contains `com.metamx.emitter.http` property set then an instance of `HttpPostEmitter` will be created.
```

Comment:
Can we make it where these all use the same workflow as the "custom" one? as in there is no difference between how a custom one is defined and how an http or logging one is defined?